app/views/devise/sessions/new.html.haml

```diff
@@ -0,0 +1,17 @@
+%h2 Sign in
+= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
+  .field
+    = f.label :email
+    %br/
+    = f.email_field :email, autofocus: true
+  .field
+    = f.label :password
+    %br/
+    = f.password_field :password, autocomplete: "off"
+  - if devise_mapping.rememberable?
+    .field
+      = f.check_box :remember_me
+      = f.label :remember_me
+  .actions
+    = f.submit "Sign in"
+= render "devise/shared/links"
```

Comment:
because it didn't where install correctly when we started the branch. and I didn't want to change it when sigu where fixing it.